### DISCUSSION TITLE
How to see the exact time of a release on github mobile webpage?

### DISCUSSION THREAD

**Discussion by oxygen-dioxide** [community/community#45965] [Repositories]
### Select Topic Area

Product Feedback

### Body

On github mobile webpage, we can only see the approximate time of a release
 like "3 weeks ago". How can we see the exact time like, "2023-1-1 12:00"
![Screenshot_20230201_080215_com microsoft emmx](https://user-images.githubusercontent.com/54425948/215912122-3444bbf3-a394-4390-8eb4-7a814cf217d3.jpg)

**Comment by jge162** [comment]
@oxygen-dioxide you will need to view the release information on the ->`desktop site.` You can access the desktop site by visiting GitHub.com on a web browser or by switching to "desktop view" on your mobile browser. (if it is supported on your phone)

**Reply by oxygen-dioxide** [reply]
I've tried desktop mode and it doesn't solve my issue. I still see approximate time on webpage. 
![Screenshot_20230201_083142_com microsoft emmx](https://user-images.githubusercontent.com/54425948/215915681-cf2bac77-564d-471e-be6c-b246b9ac4616.jpg)

**Comment by jge162** [comment]
Because normally to see the exact time, you can access the release page on a desktop or laptop computer and view the information in the release section. The exact time is listed under "Published" for each release.
